src/Analyzers/Core/Analyzers/RemoveUnnecessaryImports/AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer.cs

```diff
@@ -117,6 +117,7 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
             }
         }
 
+        public bool IsHighPriority => false;
```

Comment:
Not urgent but I wonder if this is another one to mark. Honestly if I'm ever invoking a fix on the usings block, I more or less assume it's always this one too.

Reply:
Sure. But I'll do that in a following pr. I'll open issue to track this.

Reply:
Tracking this here: https://github.com/dotnet/roslyn/issues/53705